### ISSUE TITLE
USE Large v5 backprop does not work

### ISSUE THREAD

**Issue by ivan-zidov** [tensorflow/hub#455]
`Hi there,
I have an issue with finetuning USE Large V5 (tf2.1.0rc1). I want my model to have 3 USE for 3 inputs. My 3 inputs have 3, 21 and 26 sentances.  Here is my code for model.
    
    inputs = [Input(shape=[], dtype=tf.string) for i in range(50)]
    input_category = Input(shape=(5,))
    input_host = Input(shape=(6,))

    embs_1 = [hub.KerasLayer(embed_1 trainable=True)(in_s) for in_s in inputs[:3]]
    embs_2 = [hub.KerasLayer(embed_2, trainable=True)(in_s) for in_s in inputs[3:24]]
    embs_3 = [hub.KerasLayer(embed_3, trainable=True)(in_s) for in_s in inputs[24:50]]
  
    x = Concatenate()(embs_1+embs_2+embs_3+[input_category,input_host])
    x = Dropout(0.5)(x)

    out = Dense(30,activation="sigmoid",name="out")(x)

    model = Model(inputs=[inputs,input_category,input_host], outputs=[out])
    model.compile(optimizer=Adam(), loss="binary_crossentropy")

After my model is compiled, I start training and then this happens:
![image](https://user-images.githubusercontent.com/51969305/71352323-4d366300-2576-11ea-8bba-188ac846da2b.png)
![image](https://user-images.githubusercontent.com/51969305/71352342-57586180-2576-11ea-873a-5d0e565e3dd3.png)

So model compiles successfully and it starts training process. After first batch, it crashes because it cant compute backprop. Can you help me? Thnak you

**Comment by gowthamkpr** [comment]
@ivan-zidov Can you please provide a working colab example to reproduce this issue. Thanks!

**Comment by ivan-zidov** [comment]
https://colab.research.google.com/drive/1y4WhhcnflTReWViO9iInRm5OJksS8YX2

Here, I have demonstrated bug. I have used simplified version of my original model, but I've managed to replicate bug.

**Comment by gowthamkpr** [comment]
@ivan-zidov Yes I was able to run into the error too on colab.

**Comment by gowthamkpr** [comment]
@ivan-zidov Just an update on the issue. This issue is caused in Keras. We are working to fix this issue.

**Comment by andresusanopinto** [comment]
The issue has been fixed in TF-2.2.
